src/main/java/ninjabrainbot/gui/panels/main/ThrowPanel.java

```diff
@@ -231,7 +231,7 @@ private void setThrow(IThrow t) {
 			alpha.setText(String.format(Locale.US, "%.2f", t.alpha_0()));
 			correctionSgn = Math.abs(t.correction()) < 1e-7 ? 0 : (t.correction() > 0 ? 1 : -1);
 			if (correctionSgn != 0) {
-				correction.setText(String.format(Locale.US, t.correction() > 0 ? "+%.2f" : "%.2f", t.correction()));
+				correction.setText(String.format(Locale.US, t.correction() > 0 ? "+%.3f" : "%.3f", t.correction()));
```

Comment:
Dont change existing behaviours for users who dont use this feature, showing this amount of information is redundant to them. You should only display 3 significant figures here if the use tall res setting is enabled.